packages/web/src/common/store/lineup/sagas.ts

```diff
@@ -100,6 +103,14 @@ function* filterDeletes<T extends Track | Collection>(
         return null
       }
 
+      if (
+        !isPremiumAlbumsEnabled &&
+        'playlist_id' in metadata &&
+        metadata.is_stream_gated
+      ) {
+        return null
+      }
```

Comment:
Nice! I assume this is safe to do since it's also how we handle deleted content? Main concern would be pagination, but I think it's safe.

Reply:
i think it's safe - this is how we filtered out premium content before and didn't notice any issues. i also tried scrolling to make sure the pagination was working.